swagger.json

```diff
@@ -1821,6 +1821,80 @@
             }
           }
         },
+        "/fleet/vehicles/stats": {
+              "get": {
+                  "tags": ["Fleet"],
+                  "summary": "/fleet/vehicles/stats",
+                  "description": "Fetch engine state and aux input data for all vehicles in the group between a start/end time. Data returned may be affected by device connectivity and processing time.",
+                  "operationId": "GetVehicleStats",
+                  "parameters": [
+                    {
+                      "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                      "name": "startMs",
+                      "description": "Time in Unix epoch milliseconds for the start of the query.",
+                      "required": true,
+                      "in": "query",
+                      "type": "integer"
+                    },
+                    {
+                      "name": "endMs",
+                      "description": "Time in Unix epoch milliseconds for the end of the query.",
+                      "required": true,
+                      "in": "query",
+                      "type": "integer"
+                    },
+                    { 
+                      "name": "series",
```

Comment:
should this be `filter` ?

Reply:
Renamed it to `series` as per Greg's comment earlier.

Reply:
or, should the example be `series=engineState,auxInput2`?

Reply:
changed the example